src/frontend/src/components/DynamicBackgroundContainer.jsx

```diff
@@ -9,65 +11,46 @@ const useStyles = makeStyles((theme) => ({
     backgroundSize: 'cover',
     transition: 'opacity 1s ease-in-out',
     position: 'absolute',
-    height: '100vh',
+    height: `calc(100vh - ${theme.spacing(8)}px)`,
     width: '100%',
+    opacity: '.45',
     top: 0,
   },
   child: {
     transition: 'opacity 1s ease-in-out',
   },
 }));
 
-export default function DynamicBackgroundContainer(props) {
-  const [backgroundImgSrc, setBackgroundImgSrc] = useState('');
-  const [transitionBackground, setTransitionBackground] = useState(true);
-  const classes = useStyles();
-
-  useEffect(() => {
-    async function getBackgroundImgSrc() {
-      try {
-        // Uses https://unsplash.com/collections/1538150/milkyway collection
-        /* Other Options:
-        - https://unsplash.com/collections/291422/night-lights
-        */
-
-        // Ensure we are using an image which fits correctly to user's viewspace
-        const dimensions = `${window.innerWidth}x${window.innerHeight}`;
-        const response = await fetch(`https://source.unsplash.com/collection/894/${dimensions}`);
-
-        if (response.status !== 200) {
-          throw new Error(response.statusText);
-        }
-
-        setBackgroundImgSrc(response.url);
-      } catch (error) {
-        setBackgroundImgSrc('../../images/hero-banner.png');
-        console.error(error);
-      } finally {
-        setTransitionBackground(false);
-      }
-    }
+const randomGenerator = (min, max) => {
+  return Math.floor(Math.random() * (max - min + 1)) + min;
```

Comment:
No `return` after `=>`, remove the `{...}` and return the result of the expression.